tasks/post-to-cmr/tests/cmr-test.js

```diff
@@ -128,9 +129,61 @@ test.serial('postToCMR skips CMR step if the metadata file uri is missing', asyn
     }]
   }];
 
-  const output = await postToCMR(newPayload);
+  try {
+    await postToCMR(newPayload);
+  }
+  catch (error) {
+    t.true(error instanceof CMRMetaFileNotFound);
+  }
+});
+
+test.serial('postToCMR throws an error if any granule is missing a Meta file', async (t) => {
+  const newPayload = t.context.payload;
+  const newGranule = {
+    granuleId: 'MOD11A1.A2017200.h19v04.006.2017201090555',
+    files: [{
+      filename: `s3://${t.context.bucket}/to/file.xml`
```

Comment:
So this will not pass through as a CMR metadata file because it just ends in `xml` and not `.cmr.xml`? If so, I think it might be clearer to just give this file a random name/extension, because at first glance it looks like this is a CMR XML file

Reply:
Same thing for line 128, if my understanding is correct

Reply:
I think it's a useful test condition to keep us on our toes if the implementation of `isCMRFile` ever changes.